node.ks

```diff
@@ -18,21 +20,24 @@ global nodeStageFuelInit is 0.
 // keep ship pointed at node
 sas off.
 //lock steering to lookdirup(nodeNd:deltav, ship:facing:topvector). // ORIGINAL CODE
-lock steering to OffsetSteering(lookdirup(nodeNd:deltav, ship:facing:topvector)). //LFC's mod
+lock steering to OffsetSteering(lookdirup(nodeNd:deltav, ship:up:vector)). //LFC's mod
 
 // estimate burn direction & duration
 global nodeAccel is uiAssertAccel("Node").
 global nodeFacing is lookdirup(nodeNd:deltav, ship:facing:topvector).
 global nodeDob is (nodeNd:deltav:mag / nodeAccel).
 
 uiDebug("Orient to burn").
-wait until vdot(facing:forevector, nodeFacing:forevector) >= 0.995 or nodeNd:eta <= nodeDob / 2.
+// If have time, wait to ship almost align with maneuver node.
+// If have little time, wait at least to ship face inside 40º cone from the node. This prevents backwards burns.
+wait until (vdot(facing:forevector, nodeFacing:forevector) >= 0.995) or
+           ((nodeNd:eta <= nodeDob / 2) and (vdot(facing:forevector, nodeFacing:forevector) >= 0.94)). WAIT 5.
```

Comment:
Could the hardcoded `WAIT 5` ever cause us to wait too long and miss the beginning of our burn window?